istioctl/cmd/kubeinject.go

```diff
@@ -130,12 +138,16 @@ func (e ExternalInjector) Inject(pod *corev1.Pod, deploymentNS string) ([]byte,
 			}()
 		}
 		tlsClientConfig.ServerName = fmt.Sprintf("%s.%s.%s", cc.Service.Name, cc.Service.Namespace, "svc")
-	}
-	client := http.Client{
-		Timeout: time.Second * 5,
-		Transport: &http.Transport{
-			TLSClientConfig: tlsClientConfig,
-		},
+	} else if strings.Contains(address, ".googleapis.com/") || strings.Contains(address, ".googleapis.com:443/") {
```

Comment:
nit: break this out into a function in google.go

Reply:
Done